backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/SortingHandler.java

```diff
@@ -27,23 +30,50 @@
 
     switch (names[0]) {
       case ModelDBConstants.METRICS:
-        return processKeyValueSort(names[1], ascending, "metrics");
+        return InternalFuture.completedInternalFuture(
+            processKeyValueSort(names[1], ascending, "metrics"));
       case ModelDBConstants.HYPERPARAMETERS:
-        return processKeyValueSort(names[1], ascending, "hyperparameters");
+        final var hyperparameterSoryPredicate =
+            processKeyValueSort(names[1], ascending, "hyperparameters");
+        final var configHyperparameterSoryPredicate =
+            processConfigHyperparametersSort(names[1], ascending);
+        List<QueryFilterContext> queryFilterContexts = new ArrayList<>();
+        queryFilterContexts.add(hyperparameterSoryPredicate);
+        queryFilterContexts.add(configHyperparameterSoryPredicate);
+        return InternalFuture.completedInternalFuture(
+            QueryFilterContext.combine(queryFilterContexts));
     }
 
     return InternalFuture.failedStage(new InvalidArgumentException("Sort key cannot be handled"));
   }
 
-  private InternalFuture<QueryFilterContext> processKeyValueSort(
-      String key, boolean ascending, String fieldType) {
+  private QueryFilterContext processKeyValueSort(String key, boolean ascending, String fieldType) {
     var sql =
-        "select experiment_run_id as id, kv_value as value from keyvalue where entity_name=\"ExperimentRunEntity\" and field_type=:sort_field_type and kv_key=:sort_key";
+        "select experiment_run_id as id, kv_value as value from keyvalue where entity_name=:entityName and field_type=:sort_field_type and kv_key=:sort_key";
     var queryContext =
         new QueryFilterContext()
             .addBind(q -> q.bind("sort_field_type", fieldType))
+            .addBind(q -> q.bind("entityName", "ExperimentRunEntity"))
```

Comment:
```suggestion
            .addBind(q -> q.bind("entityName", "\"ExperimentRunEntity\""))
```

Reply:
No need to wrap with `""` we are removing quotes as per @coryverta PR.

Reply:
Are you absolutely sure? I believe his PR added the quotes

Reply:
@conradoverta 's reference PR: https://github.com/VertaAI/modeldb/pull/2294/files